docs/sphinx_doc/source/tutorial/208-distribute.md

```diff
@@ -45,39 +45,35 @@ A-->C
 
 Please follow the steps below to deploy your application distributedly.
 
-### Write your agents
+### Convert your agents
 
-To use distributed functionality, your agent class needs to inherit from `RpcAgentBase`.
-`RpcAgentBase` requires several additional parameters compared to `AgentBase` during initialization.
+`AgentBase` provided the `to_dist` method to convert the agent into a distributed version.
+`to_dist` requires several parameters.
 
 - `host`: the hostname or IP address of the machine where the agent runs, defaults to `localhost`.
 - `port`: the port of this agent's RPC server, defaults to `80`.
 - `launch_server`: whether to launch an RPC server locally, defaults to `True`.
 - `local_mode`: set to `True` if all agents run on the same machine, defaults to `True`.
-
-But don't worry, `RpcAgentBase` shares the same interface as `AgentBase`, you only need to implement the `reply` method. You can even copy the `reply` method of an `AgentBase` subclass and use it in `RpcAgentBase`.
+- `lazy_launch`:  if set to `True`, only launch the server when the agent is called.
 
 ### Run in multi-process mode
 
 AgentScope supports deployment in multi-process mode, where each agent is a sub-process of the application's main process, and all agents run on the same machine.
-Its usage is very similar to single-process mode. The only difference lies in the initialization phase.
+The usage is exactly the same as single process mode, and you only need to call the `to_dist` method after initialization.
 
-Suppose you have classes `A` and `B`, both of which inherit from `RpcAgentBase`.
-You can run the application in multi-process mode by passing in an additional parameter `port`, the other parts are completely the same as the single-process mode.
+Suppose you have classes `A` and `B`, both of which inherit from `AgentBase`.
 
 ```python
 # import packages
 
 a = A(
     name="A",
     ...,
-    port=12001, # port is required, other fields like host, launch_server and local_mode use the default value
-)
+).to_dist()
```

Comment:
An issue is that if port defaults to 80,do agent `a` and `b` take the same port here? Or they are allocated with different ports automatically?

Reply:
In standalone mode, agents will automatically detect available ports

Reply:
Since multi-processing multi-machine and multi-processing single-machine mode share the same `to_dist` function, plz add more explanation to the docstring of function `to_dist`. 
For example, in single-machine mode, if the port is not provided, an automatically selected port will be used.